index.bs

```diff
@@ -1,11 +1,62 @@
-<pre class='metadata'>
-Title: Web Extensions
-Shortname: webextensions
-Level: 1
-Status: w3c/CG-DRAFT
-Group: WECG
-URL: https://w3c.github.io/webextensions
-Editor: Mukul Purohit, Microsoft Corporation https://www.microsoft.com, mpurohit@microsoft.com
-Editor: Tomislav Jovanovic, Mozilla https://www.mozilla.org/, tjovanovic@mozilla.com
-Abstract: [Placeholder] Abstract.
-</pre>
+<pre class='metadata'>
+Title: Web Extensions
+Shortname: webextensions
+Level: 1
+Status: w3c/CG-DRAFT
+Group: WECG
+URL: https://w3c.github.io/webextensions
+Editor: Mukul Purohit, Microsoft Corporation https://www.microsoft.com, mpurohit@microsoft.com
+Editor: Tomislav Jovanovic, Mozilla https://www.mozilla.org/, tjovanovic@mozilla.com
+Abstract: [Placeholder] Abstract.
+Markup Shorthands: markdown yes
+</pre>
+
+# Manifest
+
+A WebExtension must have a manifest file at its root directory.
+
+## Manifest file
+
+A manifest file is a [[!JSON]] document named `manifest.json`. Malformed JSON files are not supported.
+
+## Manifest keys
+
+If manifest keys that are not defined in this specification are specified, implementors may ignore those keys.
+
+If manifest keys that are defined in this specification are specified with a different JSON type than defined in this specification, implementors may ignore those keys.
+
+A manifest file must have these keys properly specified:
+
+* <a href="#key-name">`name`</a>: required.
+* <a href="#key-version">`version`</a>: required.
+* <a href="#key-default_locale">`default_locale`</a>: required under some conditions.
+* <a href="#key-description">`description`</a>: optional.
+* <a href="#key-icons">`description`</a>: optional.
+
+### Key `name`
+
+This key must be present. Its value must be a JSON string.
+
+It should be used as the user-facing name of the extension.
```

Comment:
Would something like this be a little clearer? I'm not sure if it's closer to normal spec language or not:

> The browser should use this to identify the extension when displaying it to a user.

Reply:
I removed this from the PR, description will need to be in a separate one.